digits/model/tasks/caffe_train.py

```diff
@@ -1031,6 +1045,38 @@ def after_runtime_error(self):
     ### TrainTask overrides
 
     @override
+    def get_task_stats(self,epoch=-1):
+        """
+        return a dictionary of task statistics
+        """
+
+        loc, mean_file = os.path.split(self.dataset.get_mean_file())
+
+        stats = {
+            "image dimensions": self.dataset.get_feature_dims(),
+            "mean file": mean_file,
+            "snapshot file": self.get_snapshot_filename(epoch),
+            "solver file": self.solver_file,
+            "train_val file": self.train_val_file,
+            "deploy file": self.deploy_file,
+            "framework": "caffe"
+        }
+
+        if hasattr(self,"original_file"):
+            stats.update({"original file": self.original_file})
```

Comment:
Internally in the DIGITS tree, let's keep referring to this file as "original". But since https://github.com/BVLC/caffe/pull/3863 has been merged, I'm hopeful that we'll be able to eventually provide all-in-one networks as outputs. So I vote for "network file" here, so that when we remove "train_val file" and "deploy file" later the name of this field doesn't have to change.